cppForSwig/cryptopp/Makefile

```diff
@@ -36,7 +37,6 @@ IS_SUN_CC = $(shell $(CXX) -V 2>&1 | $(EGREP) -c "CC: Sun")
 GAS210_OR_LATER = $(shell echo "" | $(AS) -v 2>&1 | $(EGREP) -c "GNU assembler version (2\.[1-9][0-9]|[3-9])")
```

Comment:
If possible, I'd prefer that the Crypto++ make system be left alone for a couple of reasons.

- At my last job, we used third-party packages and were basically told to leave the code alone whenever possible. Our build system would just kick off whatever it was given and leave it at that, unless there was a major problem that had to be addressed, specific values that had to be modified, etc. The idea of redoing a third-party Makefile, no matter how messy it may be, just seems like asking for trouble.
- AFAIK, we're going to drop Crypto++ as soon as is feasible. Wei has seemingly abandoned it, and it's not fun to have to patch it every time a new version of GCC comes out.